common/lib/xmodule/xmodule/course_module.py

```diff
@@ -23,14 +22,14 @@
 from .fields import Date
 from django.utils.timezone import UTC
 
+from xmodule import course_metadata_utils
```

Comment:
Why not just import the functions that are used? Also, this needs to be moved up with the other xmodule imports.